sdk/cosmosdb/cosmos/src/globalEndpointManager.ts

```diff
@@ -40,15 +40,20 @@ export class GlobalEndpointManager {
 
   public preferredLocationsCount: number;
   /**
-   * Flag to enable/disable the Per Partition Level Failover (PPAF)
+   * Flag to enable/disable the Per Partition Level Failover (PPAF). Contains the value from the Connection policy.
    * @internal
    */
-  public enablePartitionLevelFailover: boolean;
+  private enablePartitionLevelFailover: boolean;
   /**
-   * Flag to enable/disable the Per Partition Level Circuit Breaker (PPCB)
+   * Cached PPAF enablement status from the last account refresh
    * @internal
    */
-  public enablePartitionLevelCircuitBreaker: boolean;
+  public lastKnownPPAFEnabled?: boolean;
```

Comment:
this name is misleading; can we rename it capture the idea? Also, why it's optional?